toolkit/Makefile

```diff
@@ -132,20 +132,25 @@ PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR
 REPO_LIST          ?=
 SRPM_URL_LIST      ?= https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/base/srpms
 
-# Preview URLs/repos will be appended (via override) even if the user sets custom values for the lists.
-ifeq ($(USE_PREVIEW_REPO),y)
-   override PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/preview/base/$(build_arch)
-   override PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/preview/base/debuginfo/$(build_arch)
-   override PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/preview/Microsoft/$(build_arch)
-   override SRPM_URL_LIST      += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/preview/base/srpms
-   ifneq ($(wildcard $(PREVIEW_REPO)),)
-      override REPO_LIST += $(PREVIEW_REPO)
-   else
-      $(warning )
-      $(warning ######################### WARNING #########################)
-      $(warning 'USE_PREVIEW_REPO=y' set but '$(PREVIEW_REPO)' is missing. Regenerate toolkit's 'repos' directory. Remove 'USE_PREVIEW_REPO' for core builds.)
-      $(warning ######################### WARNING #########################)
-      $(warning )
+ifneq ($(DAILYBUILDID),)
+   $(warning "hello")
+   override PACKAGE_URL_LIST   := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILYBUILDID)-amd64/built_rpms_all
+   override SRPM_URL_LIST      := https://mariner3dailydevrepo.blob.core.windows.net/daily-repo-$(DAILYBUILDID)-amd64/SRPMS
```

Comment:
Do we need/want to inject a repo here? Is this just a blob of rpms, or is there an actual repo xml behind it? We could make a little `.sh` tool that spits out a `.repo` file into `./build/dailyrepo.rep` or something based on the ID, and add to `REPO_LIST`.

Reply:
It's a real repo.  The mariner-official-repos.rpm in the Sean-BlobStore-Repo knows where the repo lives.  So when the chroot is built from the various components it donwloads and installs the repo into the chroot correctly.  

So...I'm not sure if it's needed, but I'll defer to your thoughts on that.

Reply:
Ok, if its getting side-loaded into the chroot that also works (not sure what the mechanism there is).